exercises/practice/palindrome-products/.meta/test_template.erb

```diff
@@ -16,7 +16,7 @@ class PalindromesTest < Minitest::Test
                 palindromes = Palindromes.new(max_factor: <%= cases["input"]["max"] %> <%= cases["input"]["min"] != 1 ?  ", min_factor: #{cases["input"]["min"]}" : "" %>)
                 palindromes.generate
                 actual = palindromes.<%= cases["property"] %>.value, palindromes.<%= cases["property"] %>.factors
-                expected = <%= cases["expected"]["factors"].size > 1 ? "#{cases["expected"]["factors"].permutation(2).to_a.map{|factors| [cases["expected"]["value"], factors]}}" : "#{cases["expected"]["value"] ? cases["expected"]["value"] : "nil"}, #{cases["expected"]["factors"]}"%>
+                expected = <%= cases["expected"]["factors"].size > 1 ? "#{ cases["expected"]["factors"].permutation(2).to_a.map{|factors| [cases["expected"]["value"], factors].to_s}.join(", ")}" : "#{cases["expected"]["value"] ? cases["expected"]["value"] : "nil"}, #{cases["expected"]["factors"]}"%>
```

Comment:
No space inside delimiter for interpolation.  Will not cause a problem, but looks odd since it is "imbalanced".  However, space after method name and block.  Space inside block delimiter (before the block argument list delimiter, and space inside ending of the block delimiter.

This may be fixed in a post generation process if we are using something like a linter, though.  But we can and should address it in the generator.